packages/atoms/src/TipToast.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import { Icon } from '@trezor/icons';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+
+import { Box } from './Box';
+import { Text } from './Text';
+
+type Props = {
+    content: string;
+    onClose: () => void;
+};
+
+const tipToastStyle = prepareNativeStyle(utils => ({
+    flexDirection: 'row',
+    justifyContent: 'space-between',
+    backgroundColor: utils.colors.black,
+    color: utils.colors.white,
+    borderRadius: utils.borders.radii.small,
+    padding: utils.spacings.small,
+}));
+
+const iconWrapperStyle = prepareNativeStyle(utils => ({
+    justifyContent: 'center',
+    alignItems: 'center',
+    borderRadius: utils.borders.radii.small,
+    backgroundColor: utils.colors.gray800,
+    width: 40,
+    height: 40,
+}));
+
+export const TipToast = ({ content, onClose }: Props) => {
+    const { applyStyle } = useNativeStyles();
+
+    return (
+        <Box style={applyStyle(tipToastStyle)}>
+            <Box>
+                <Text color="gray400" variant="hint">
+                    TIP
+                </Text>
+                <Text color="white" variant="hint">
+                    {content}
```

Comment:
Content doesn't evoke ReactNode (which accepts string anyway) to me. Maybe if content is not explicit enough, I would choose the `title / description` duo. Since the "TIP" text will not be static I guess, that would make sense, right?

Reply:
`title` is good enough to me :) The main problem I have with `content` is that I imagine something more complex and more structured than just plain text.